### PR TITLE
enable TGL-H build support

### PR DESCRIPTION
**Pull request by xiulipan**
- change FW output name to sof-tgl-h.ri to match Linux driver
- Add tgl-h support in scripts/xtensa-build-all.sh
- update the rimage submodule to latest upstream.

Need rimage update https://github.com/thesofproject/rimage/pull/37 for the tgl-h rimage support.

fix https://github.com/thesofproject/sof/issues/3666

### CODE REVIEW

**Review by lgirdwood — APPROVED** [review]

File: scripts/xtensa-build-all.sh [hunk 1]
```diff
@@ -255,6 +255,19 @@ do
 				PRIVATE_KEY_OPTION="-D${SIGNING_TOOL}_PRIVATE_KEY=$pwd/keys/otc_private_key_3k.pem"
 			fi
 			;;
+		tgl-h)
+			PLATFORM="tgph"
```
Comment (aiChaoSONG) [inline scripts/xtensa-build-all.sh:259]: tgph?

**Comment by lgirdwood** [comment]
@xiulipan can you check CI, it's failing for the TGL_H platform you have just added (unless this is also pending for CI). @zrombel I assume CI is good for TGL-H (or is there a dependency ?)

**Comment by zrombel** [comment]
@lgirdwood QB CI is building TGL_H at this point with success but this PR is breaking this build.
Here is the error
```
cd /quickbuild/workspace1/24733/SOF_FW/build_fw/src/arch/xtensa && ../../../rimage_ep/build/rimage -o sof-tgl-h.ri -c /quickbuild/workspace1/24733/SOF_FW/rimage/config/tgl-h.toml -s 1344 -k /quickbuild/workspace1/24733/SOF_FW/keys/otc_private_key.pem -i 3 -f 1.5 -b 1 -e -x 24 bootloader-tgl-h sof-tgl-h
error: can't open '/quickbuild/workspace1/24733/SOF_FW/rimage/config/tgl-h.toml' file
```
Currently working command looks as follows:
```
cd /quickbuild/workspace1/24733/SOF_FW/build_fw/src/arch/xtensa && ../../../rimage_ep/build/rimage -o sof-tgl.ri -c /quickbuild/workspace1/24733/SOF_FW/rimage/config/tgl.toml -s 1344 -k /quickbuild/workspace1/24733/SOF_FW/keys/otc_private_key.pem -i 3 -f 1.5 -b 1 -e -x 24 bootloader-tgl sof-tgl
```

So there is a problem with toml file.

**Comment by xiulipan** [comment]
@lgirdwood @zrombel I add a TODO for rimage update also.
I have updated the rimage submodule to latest upstream version and hope this issue can be solved.

**Comment by zrombel** [comment]
@xiulipan After rimage update everything builds properly :)

**Comment by xiulipan** [comment]
@zrombel Thanks for checking, now we can have sof-tgl-h.ri instead of sof-tgl.ri for tgph config.
@lgirdwood I think we are good to merge this to have more support for TGL-H now.

**Pull request merged**